spacewalk/uyuni-setup-reportdb/bin/uyuni-setup-reportdb

```diff
@@ -13,6 +13,10 @@ help() {
     echo "    --password <password>        Password for the database user"
     echo "    --autogenpw                  Auto generate a password for the database user"
     echo "    --standalone                 Configure the database server independent of Uyuni or SUSE Manager Server Database"
+    echo "    --external-host              The database host when configuring the report database using an external database server"
```

Comment:
I think you can compute this. If external-user is set, you should assume that things are external.

Reply:
yes but unless I am missing something the host is not passed in as it assumes localhost.  Somehow I need to get the "remote host" passed in.  Do you have a better suggestion @mcalmer

Reply:
or is the point rather just rename it to --host?

Reply:
i have this working but using the host